readinglist/resource.py

```diff
@@ -176,6 +176,7 @@ def collection_post(self):
         new_record = self.request.validated
         new_record = self.process_record(new_record)
         self.record = self.db.create(record=new_record, **self.db_kwargs)
+        self.request.response.status = '201 Created'
```

Comment:
I am not sure why we want to mess with status messages in our code base. This is to be dealt by the framework. We shoud set a 201 code and the thing should set the status automatically